pkg/resourcemanager/controller/networkpolicy/reconciler.go

```diff
@@ -359,6 +359,14 @@ func (r *Reconciler) reconcileIngressFromWorldPolicy(ctx context.Context, servic
 	return err
 }
 
+func portAndProtocolOf(ports []networkingv1.NetworkPolicyPort) []string {
+	var result []string
+	for _, v := range ports {
+		result = append(result, fmt.Sprintf("%v/%v", *v.Protocol, v.Port))
```

Comment:
Just to be on the safe side, do we need the following (or should we rather assume protocol to always be set)?

```suggestion
		if v.Protocol == nil {
			result = append(result, fmt.Sprintf("%v", v.Port))
		} else {
			result = append(result, fmt.Sprintf("%v/%v", *v.Protocol, v.Port))
		}
```

Reply:
Thank you! PTAL.